### ISSUE TITLE
[Bug Report] Field input label does not "break" outline

### ISSUE THREAD

**Issue by kumold** [vuetifyjs/vuetify#9413]
### Environment
**Vuetify Version:** 2.1.5
**Vue Version:** 2.6.10
**Browsers:** Edge 79.0.294.1
**OS:** Windows 10

### Steps to reproduce
1. Check checkbox
2. Appeared first input with label and placeholder have border over label text

### Expected Behavior
border should "break" (go under?) of the label

### Actual Behavior
Border goes "through" label

### Reproduction Link
<a href="https://codepen.io/kumold/pen/vYYyywx?&editable=true&editors=101" target="_blank">https://codepen.io/kumold/pen/vYYyywx?&editable=true&editors=101</a>

**Comment by 185driver** [comment]
It's not a fix, but in playing with your codepen, I found a workaround:
``` html
<v-text-field
  :placeholder="is_visible && 'assbsd'"
  label="Regular"
  outlined
></v-text-field>
```

**Comment by kumold** [comment]
> It's not a fix, but in playing with your codepen, I found a workaround:
> 
> ```
> <v-text-field
>   :placeholder="is_visible && 'assbsd'"
>   label="Regular"
>   outlined
> ></v-text-field>
> ```

Thanks) That'll do for now.

**Comment by rflament-icabbi** [comment]
I have a similar issue when text field is in a menu : https://codepen.io/rflament/pen/pooRJKV

**Comment by PikminGuts92** [comment]
I expanded on the workaround solution from @185driver. Binding a boolean value to the placeholder prop causes Vuetify to throw an invalid type warning in the console.

I added a computed value to return a string value based on visibility.
```js
computed: {
  labelPrefix: function () {
    return this.is_visible ? 'show' : ''
  }
}
```

Then I bind that value to the label. Using the magic of JS logical operators, the binded value will either be empty or `Regular`.

Note: I moved the binding to the label prop to hide the animation of the label value moving from the placeholder's position in the text field to the label's position on the outline.
```html
<v-text-field
  placeholder="assbsd"
  :label="labelPrefix && 'Regular'"
  outlined
></v-text-field>
```

[https://codepen.io/pikminguts92/pen/NWWdewK](https://codepen.io/pikminguts92/pen/NWWdewK)
